app/models/conversion_host.rb

```diff
@@ -4,4 +4,19 @@ class ConversionHost < ApplicationRecord
   acts_as_miq_taggable
 
   belongs_to :resource, :polymorphic => true
+  has_many :service_template_transformation_plan_tasks, :dependent => :nullify
+
+  def active_tasks
+    service_template_transformation_plan_tasks { active }
```

Comment:
You have to define the scope in miq_request_task:
```
scope :active, -> { where(:state => "active") }
```

Then here it will look like:
```
has_many :active_tasks, -> { active }, :class_name => ServiceTemplateTransformationPlanTask
```

Reply:
Tried it but it fails during tests.